include/amqp_client_internal.hrl

```diff
@@ -35,3 +35,6 @@
      {<<"consumer_cancel_notify">>,       bool, true},
      {<<"connection.blocked">>,           bool, true},
      {<<"authentication_failure_close">>, bool, true}]).
+
+-define(CALL_TIMEOUT,
+        rabbit_misc:get_env(amqp_client, gen_server_call_timeout, 30000)).
```

Comment:
Since this is on the hot code path, we should probably do [what we do in the server](https://github.com/rabbitmq/rabbitmq-common/blob/ceec9bd6109eebbeddc9d4706ce1d71be815d7c2/src/rabbit_channel.erl#L415): in process caching of the timeout value, since all `rabbit_misc:get_env/3` (`application:get_env*`) requests go through a single gen_server.